manager/apis/app/v1alpha1/cluster_selector.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterSelector is a label query over a set of resources in the specified cluster.
+type ClusterSelector struct {
```

Comment:
I thought @roee88 wanted two selectors in the spec rather than combining them into a single structure, but it could very well be that I didn't understand him correctly.  @roee88 is this what you wanted?

Reply:
No it's fine as is. I would just name `ClusterSelector` differently because it's not just a cluster selector. Maybe even just `Selector`.

Reply:
Renamed.